docs/Plugins/webhook.md

```diff
@@ -4,50 +4,35 @@ description: >
   Webhook Plugin
 ---
 
-## Overview 
+## Summary
 
-Incoming Webhooks are your solution to bring data to Apache DevLake when there isn't a specific plugin ready for your DevOps tool. An Incoming Webhook allows users to actively push data to DevLake. 
+Incoming Webhooks are your solution to bring data to Apache DevLake when there isn't a specific plugin ready for your DevOps tool. An Incoming Webhook allows users to actively push data to DevLake.
 
 When you create an Incoming Webhook within DevLake, DevLake generates a unique URL. You can then post JSON payloads to this URL to push data directly to your DevLake instance.
 
 In v0.14+, users can push "incidents" and "deployments" required by DORA metrics to DevLake via Incoming Webhooks.
 
-## Creating webhooks in DevLake
+## Configuration
```

Comment:
1. Keep `Entities` and `Metrics` sections.
2. Add a column `Incoming Webhook` in the `supported date source` doc